### PR TITLE
Fixed some secondary nav styling issues

### PR DESCRIPTION
**Pull request by mmmavis**
Related to #1465

@kristinashu ping me on Slack when you wanna review. I'll start a local server for you do to live QA.

### CODE REVIEW

File: network-api/networkapi/wagtailpages/templates/wagtailpages/tags/primary_page_menu.html [hunk 1]
```diff
@@ -1,12 +1,21 @@
...
 
 {% if singleton_page == False %}
-<div class="hidden-sm-down d-flex flex-row multipage-nav primary-menu" id="multipage-nav">
-	{% for entry in menu_pages %}
-	<div class="mx-3">
-	  <a class="multipage-link {% if current.title == entry.page.title %}multipage-link-active{% endif %}" href="{% pageurl entry.page %}">{% if entry.depth == 2 %}→ {% endif %}{{ entry.menu_title }}
-	  </a>
-	</div>
-	{% endfor %}
+<div class="container hidden-sm-down my-4">
```
Comment (mmmavis) [inline network-api/networkapi/wagtailpages/templates/wagtailpages/tags/primary_page_menu.html]: tidy up DOM structure here so it aligns with fellowship minisite's (designers like fellowships' nav better)

**Comment by mmmavis** [comment]
@gvn let me know if you want me to ngork so you can review...

File: network-api/networkapi/wagtailpages/templates/wagtailpages/tags/primary_page_menu.html (hunk 1 above)
Comment (ScottDowne) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/tags/primary_page_menu.html]: Just a heads up, but some of the alignment on the primary nav may be changing a bit. https://github.com/mozilla/foundation.mozilla.org/pull/1490

Hopefully it's not changing anything that you need to worry about here.

Comment (mmmavis) [reply network-api/networkapi/wagtailpages/templates/wagtailpages/tags/primary_page_menu.html]: cool thanks for the headsup @ScottDowne

**Pull request merged**